### DISCUSSION TITLE
Oral History (as) Data and other CollectionBuilder tools

### DISCUSSION THREAD

**Discussion by eagandean** [CollectionBuilder/collectionbuilder.github.io#89] [Q&A] [answered]
Hello, I'm looking for options to digitize an existing oral history archive. I noticed that the Idaho Queered site uses both the transcript coding/browsing tools from the Oral History (as) Data project and some of the mapping and visualization tools from the greater CB repertoire. I've found discrete documentation for each project/platform, but is there a way to find documentation for using these tools together? Also, is this integration possible for the new OHD? Thank you!

**Comment by dcnb** [comment, marked as answer]
Hi @eagandean -- we've done this in a couple different ways, so there are definitely ways to combine the two platforms. There is not any documentation for using them together right now, although we are trying to figure out how to integrate the features better. Feel free to reach out to me and let me know what you're thinking at my work email dbecker@uidaho.edu and I can talk you through what needs to happen. I've been working recently with a librarian from Tacoma on this --> [https://uwtacomalibrary.github.io/TJLS](https://uwtacomalibrary.github.io/TJLS)

There might best some good overlap for you with that template.

**Reply by eagandean** [reply]
Thank you so much! The Tacoma project looks wonderful. I will definitely reach out if my org approves DB! Thanks for your quick reply and hard work.
